Ports/dropbear/package.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash ../.port_include.sh
+#!/usr/bin/env bash ../.port_include.sh
```

Comment:
Does it actually work to specify multiple arguments in shebangs?

Reply:
I remember running into this when changing Serenity's `env`. Some `env` implementations allow that by default and others require a `-S` flag or something like that.

Reply:
I see, so the kernel passes everything following the initial binary name as a single argument (in this case, `/usr/bin/env "bash ../.port_include.sh"`), and the `env` implementation can then optionally support splitting this as a shell would otherwise do it.

Reply:
Sorry, I'm not quite following - is there a change you'd like me to do here?

Reply:
You *might* need to add `-S` as @asynts says

Reply:
Updated

Reply:
I think using `-S` breaks support for OpenBSD: https://man.openbsd.org/env.1

Reply:
So no `-S`? 😅

Reply:
@jonathanturner: I don't think it would work without.